tf2onnx/tfonnx.py

```diff
@@ -1676,6 +1705,7 @@ def where_op(ctx, node, name, args):
     "BiasAdd": (biasadd_op, []),
     "BiasAddV1": (biasadd_op, []),
     "Cast": (cast_op, []),
+    "CheckNumerics": (identity_op, ["Identity"]),
```

Comment:
@pengwa , according to tf [spec](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/check-numerics), it will throw if there is Nan or Inf, otherwise, pass the tensor through.
for onnx, seems there is no equivalent like Assert, and we cannot really tell whether there is Nan/Inf at converting stage.
so seems all we can do it is map to Identity.

Reply:
@lucienwang1009, is this from user script or some TF api call?

Reply:
it's in user script, nevertheless this op won't affect the model output.